tests/FSharp.Test.Utilities/XunitHelpers.fs

```diff
@@ -90,6 +90,14 @@ module TestCaseCustomizations =
         else
             testCase.TestMethod
 
+    let sha = Security.Cryptography.SHA256.Create()
+
+    let addBatchTrait (testCase: ITestCase) =
+        let data = Text.Encoding.UTF8.GetBytes testCase.DisplayName
+        let hashCode = BitConverter.ToUInt32(sha.ComputeHash(data), 0)
+        let batch = hashCode % 4u + 1u
```

Comment:
Why `% 4` here?

Reply:
just to get a number from   1  to  4. this is hardcoded just for now.

Reply:
I wanted to get the batch number from a custom hashCode that is stable between executions. I heard bad things about String.GetHashcode() in this regard.